fetch.js

```diff
@@ -20,7 +20,10 @@ var argv = require('yargs')
   .alias('h', 'help')
```

Comment:
Putting this here as I'm not sure where it's actually coming from. When doing a full test like `node index.js`, these errors start to pop up, but not when testing individual sources. We can obviously disable the limit, but this probably points to something not great happening.

```
(node:5540) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 error listeners added. Use emitter.setMaxListeners() to increase limit
```

Reply:
Not really. That's just a message indicating more than 11 muxed stream - its safe to ignore, but I'll check it out.

Reply:
I published a new version of scramjet that adds control to the listener number. I'll update the package.json here tomorrow and we'll see if it solved all of those.